examples/gan/_gan_mnist_pl/pl.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
```

Comment:
@rb-determined-ai If we include the original file from Pytorch Lightning, do we need to include their copyright info here?